src/Microsoft.OpenApi/Validations/ValidationRuleSet.cs

```diff
@@ -134,6 +134,28 @@ public void Add(ValidationRule rule)
             item.Add(rule);
         }
 
+        /// <summary>
+        /// Remove a rule by name.
+        /// </summary>        
+        /// <param name="ruleName">Name of the rule.</param>
+        public void Remove(string ruleName)
```

Comment:
Having an overload that allows people to provide element type as well might be useful

Reply:
added